python/ray/rllib/utils/tf_run_builder.py

```diff
@@ -47,7 +47,8 @@ def get(self, to_fetch):
                 self._executed = run_timeline(
                     self.session, self.fetches, self.debug_name,
                     self.feed_dict, os.environ.get("TF_TIMELINE_DIR"))
-            except Exception:
+            except Exception as ex:
+                logger.info(ex)
```

Comment:
logger.exception()?

Also, I thought that in Python the cause of an exception is shown along with a newly raised exception. So you should see "ValueError"... with "Exception" above somewhere in the trace? Is that not what you're seeing?

Reply:
I resolved the conflicts and modified according to your suggestions. Without sth like `logger.exception(...)`, in our case (described in the related issue), there is no information that is helpful for exactly locating the causes.